server/src/services/Configuration.js

```diff
@@ -0,0 +1,5 @@
+export class Configuration {
+  get nodeEnv() {
+    return (process.env.NODE_ENV || "") === "" ? "development" : process.env.NODE_ENV
+  }
+}
```

Comment:
I noticed that we also have a `getNodeEnv.js` service function in the config folder, should I just use that instead?